README.md

```diff
@@ -3,18 +3,27 @@
 [![Build Status](https://api.travis-ci.org/mongodb/mongo-php-driver.png?branch=master)](https://travis-ci.org/mongodb/mongo-php-driver)
 [![Coverage Status](https://coveralls.io/repos/mongodb/mongo-php-driver/badge.svg?branch=master&service=github)](https://coveralls.io/github/mongodb/mongo-php-driver?branch=master)
 
-The purpose of this driver is to provide a thin glue between MongoDB
-and PHP, implementing only fundamental and performance-critical components
+This is the low-level PHP driver for MongoDB. It implements the same interface
+as the HHVM driver for MongoDB. The documentation for both of them is the same,
```

Comment:
This sounds like a chicken and egg problem. The PHP driver isn't implementing the HHVM interface as this suggests